components/tree-select/style/status.less

```diff
@@ -0,0 +1,46 @@
+@import (reference) '../../style/themes/index';
```

Comment:
Same above, tree select is based on the `select` component for status setting.
Please refer to these codes([cascader](https://github.com/NG-ZORRO/ng-zorro-antd/pull/7452/files) / [input-number](https://github.com/NG-ZORRO/ng-zorro-antd/pull/7462/files)) to see how to implement the status setting.